app/controllers/application_letters_controller.rb

```diff
@@ -57,6 +57,6 @@ def set_application
 
     # Only allow a trusted parameter "white list" through.
     def application_params
-      params.require(:application_letter).permit(:motivation, :event_id, :status)
+      params.require(:application_letter).permit(:motivation, :user_id, :event_id, :status)
```

Comment:
My guess is, that you shoulndn't be able to create a letter with a manually set user. instead, in the create function, the user_id should be set by the backend itself.